src/components/App/App.js

```diff
@@ -0,0 +1,36 @@
+
+import styled from 'styled-components';
```

Comment:
Adding "macro" to the import source will add a class name to the styled Elements upon DOM render, making them better for humans to read :)
```suggestion
import styled from 'styled-components/macro';
```